js/infobar.js

```diff
@@ -253,6 +270,18 @@
     InfoBar.prototype.oncancel = {};
 
     /**
+     * Attempts to store the disabled status of the specified InfoBar in sessionStorage
+     * @param {string} prefName - The stored preference name for this InfoBar
+     */
+    InfoBar.prototype.setDisabledStatus = function(prefName) {
+        try {
+            sessionStorage.setItem(prefName, 'true');
+        } catch (ex) {
+            console.log('oncancel: Error while setting didabled status of InfoBar in sessionStorage');
```

Comment:
You are thinking just throw the error or, just swallow it. I was thinking it could be useful if/when a bug is logged to have this kind of output in the console. With that said though, Alex also has a bit of a concern that, when bundled up and minified, these might cause problems so, I am open to either of the above.